pkg/component/observability/logging/vali/vali.go

```diff
@@ -347,6 +349,15 @@ func (v *vali) getVPA() *vpaautoscalingv1.VerticalPodAutoscaler {
 func (v *vali) getIngress(secretName string) *networkingv1.Ingress {
 	pathType := networkingv1.PathTypePrefix
 
+	serviceName := valiServiceName
+	endpoint := valiconstants.PushEndpoint
+	var port int32 = kubeRBACProxyPort
```

Comment:
Nit: what about changing the type of the `kubeRBACPort` to be `int32` in the declaration above, so that you can directly do
```suggestion
        var (
	        pathType = networkingv1.PathTypePrefix

	        serviceName = valiServiceName
	        endpoint = valiconstants.PushEndpoint
	        port = kubeRBACProxyPort
	)

```

Reply:
Thank you, will do that